README.md

```diff
@@ -1,2 +1,314 @@
-# tutorial
-Renovate Bot Tutorial source
+# Renovate - Hands On Tutorial
+
+
+## Introduction
+
+Welcome to the Renovate hands-on tutorial.
+
+This tutorial is based on the Renovate GitHub App. you can also run Renovate as a CLI tool or a self-hosted application.
+
+> **Note**
+> Although this tutorial is based in the GitHub App, the concepts discussed apply to all environments.
+
+In this tutorial, you will learn how to configure Renovate and become familiar with some of the basic features.
+
+What you will learn: 
+1) Installation
+2) Onboarding
+3) Getting to know Renovate’s update PRs
+4) Dependency Dashboard 
```

Comment:
```suggestion
- 1 Installation
- 1 Onboarding
- 1 Getting to know Renovate’s update PRs
- 1 Dependency Dashboard 
```

I learned a new list syntax today. 😉 Let's stick to the Renovate docs style, as that keeps it easy to work on both repositories.

Reply:
hi @HonkingGoose ,
the new list syntax doesn't show properly on the actual document.
any idea why?
should i revert it?

Reply:
I'll go take a look. 😉

Reply:
See PR: #4.